GitUI/CommandsDialogs/FormCommit.Designer.cs

```diff
@@ -1140,7 +1156,7 @@ private void InitializeComponent()
             this.CommitAndPush.Size = new System.Drawing.Size(171, 26);
             this.CommitAndPush.TabIndex = 9;
             this.CommitAndPush.TabStop = false;
-            this.CommitAndPush.Text = "C&ommit && push";
+            this.CommitAndPush.Text = "Commit && &push";
```

Comment:
Changes like this look very sneaky. The PR's description briefly mentions this change, but it doesn't go into details as to what changed.
It won't be an easy/obvious bisect to find when and what changed, if users complain.

Please extract the changes to mnemonics in a separate commit and clearly describe the change.

Reply:
Moved the submarines to dedicated PRs. In the future, I'll remember that mixing things - although related - makes it more difficult to review and to bisect.
Unfortunately, there's little support from Git developing things in parallel in the same working copy.